app/views/coronavirus_form/check_your_answers/sections/_accommodation.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= render "coronavirus_form/check_your_answers/helpers/question_answers",
+  question_title: t("coronavirus_form.check_your_answers.sections.accommodation.title"),
+  question_items: question_items("accommodation"),
+  question_href: "accommodation",
+  detail_title: t("coronavirus_form.check_your_answers.details"),
+  detail_items: accommodation_items,
+  detail_href: "rooms-number"
```

Comment:
I wonder if this hash could be returned the `accommodation_items` in the helper, rather than being created in the view? The question applies goes to the hashes being passed into the other partials.

Reply:
This is outdated 😺